common/components/controllers/AboutProjectController.jsx

```diff
@@ -129,6 +129,7 @@ class AboutProjectController extends React.PureComponent<{||}, State> {
             <div className="col">
               <h2 className="form-group subheader">PROJECT DETAILS</h2>
               <div className="Text-section" style={{whiteSpace: "pre-wrap"}}>
+                {project && project.project_short_description ? project.project_short_description + "\n\n" : null}
```

Comment:
I think we could use a separate heading here, matching with the form field labels. So a "Short Description" and a "Project Description", while keeping them both under the Project Details H2 like they are now. Right now with a live project where you have both it's not clear which is which.

Reply:
I put them under separate 'PROJECT SUMMARY' and 'PROJECT DETAILS' for now.